classes/forms.php

```diff
@@ -636,9 +636,11 @@ public static function create_form( $newform ){
 		$added = self::save_to_db( $newform, 'primary' );
 		if( ! $added ){
 			return false;
-
 		}
 
+		// Fixes https://github.com/CalderaWP/Caldera-Forms/issues/3455
+		self::clear_cache();
```

Comment:
Is this change relevant to the PR, or is this change a stowaway?

Reply:
It is a stoway from #3456 therefore this should NOT get merged before #3456 

I need to be more careful.

Reply:
#3456 is showing as merged - resolving this conversation.